### ISSUE TITLE
Address insufficient contrast of white text over beige background: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#252]
e.g., in event date/time/location heading blocks on events listing pages.

> The color #FFF with background color #D4B982 has a contrast ratio of 1.9:1, which fails for all sized text.

![screen shot 2018-04-24 at 4 01 02 pm](https://user-images.githubusercontent.com/263285/39214005-b847adde-47d8-11e8-856f-88fdb9df1573.png)

Traceability: [Microassist Report 1 > WCAG Compliance > Color Contrast](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.fg49504ke5mr)

WCAG 2.0: [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html): The visual presentation of texts and images of text has a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. 14 point and bold or larger, or 18 point and larger. (Level AA)

**Comment by paregorios** [comment]
Change background color to #62232b for the following elements in the events listing:

 - ```#portal-columns table thead tr th, #content table thead tr th```
 - ```*portal-columns .event-listing .event-result .event-result-header, #content .event-listing .event-result .event-result-header```

Page should look like this:

![screen shot 2018-12-03 at 10 02 19 am](https://user-images.githubusercontent.com/263285/49385235-8ccbac00-f6e2-11e8-8b2e-fb423ad60160.png)

**Comment by alecpm** [comment]
This is live on staging (whenever it's back online)
